### PR TITLE
BOAC-94, header search and cohort browse dropdown

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-94

image

For now, name is UID. That will change.

### CODE REVIEW

File: boac/api/user_controller.py [hunk 1]
```diff
@@ -33,6 +33,12 @@ def user_profile():
     })
 
 
+@app.route('/api/students/all')
+def all_students():
+    sort_by = request.args['sortBy'] if 'sortBy' in request.args else None
+    return tolerant_jsonify(TeamMember.all_athletes(sort_by=sort_by))
```
Comment (johncrossman) [inline boac/api/user_controller.py]: For now, in BOAC-land, all_students is equivalent to all_athletes

File: boac/templates/index.html [hunk 2]
```diff
@@ -12,18 +12,23 @@
...
     <link href="/static/lib/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
     <link href="/static/lib/fontawesome/css/font-awesome.css" rel="stylesheet">
+    <link href="/static/lib/oi.select/dist/select.min.css" rel="stylesheet">
   </head>
 
   <body ui-view>
     <!-- Third-party: Warning, if the order below is wrong then the app will break. -->
     <script src="/static/lib/angular/angular.min.js"></script>
     <script src="/static/lib/angular-bootstrap/ui-bootstrap.js"></script>
+    <!--
```
Comment (raydavis) [inline boac/templates/index.html]: Was this a stray, @johncrossman ?

Comment (johncrossman) [reply boac/templates/index.html]: Thx, @raydavis  Nixed.

**Comment by raydavis** [comment]
UX marches on.

**Pull request merged**